ui/js/models/resultsets_store.js

```diff
@@ -272,6 +274,28 @@ treeherder.factory('ThResultSetStore', [
             return key;
         };
 
+        var getRunnableJobs = function(repoName, resultSet) {
```

Comment:
this might be better named as ``addRunnableJobs`` since it doesn't return the jobs themselves, but takes the action of adding them into the joblist.